src/Okta.Idx.Sdk.UnitTests/IdxClientLayer1Should.cs

```diff
@@ -0,0 +1,2711 @@
+            remediationProceedRequest.SetProperty("rememberMe", false);
+
+            // Proceed with "identifier"
+            var identifyProcessedResponse = await identifyRemediationOption.ProceedAsync(remediationProceedRequest);
+
+            identifyProcessedResponse.Should().NotBeNull();
+            identifyProcessedResponse.StateHandle.Should().NotBeNullOrEmpty();
+            identifyProcessedResponse.Version.Should().NotBeNullOrEmpty();
+            identifyProcessedResponse.ExpiresAt.Value.Should().Be(DateTimeOffset.Parse("2020-10-19T19:32:24.000Z"));
+            identifyProcessedResponse.Intent.Should().Be("LOGIN");
+
+            identifyProcessedResponse.Remediation.GetRaw().Should().NotBeNullOrEmpty();
+            identifyProcessedResponse.Remediation.Type.Should().Be("array");
+
+            identifyProcessedResponse.Remediation.RemediationOptions.Should().NotBeNullOrEmpty();
+            identifyProcessedResponse.Remediation.RemediationOptions.FirstOrDefault().Rel.Should().Contain("create-form");
+            identifyProcessedResponse.Remediation.RemediationOptions.FirstOrDefault().Name.Should().Be("select-authenticator-authenticate");
+            identifyProcessedResponse.Remediation.RemediationOptions.FirstOrDefault().Href.Should().Be("https://foo.okta.com/idp/idx/challenge");
+            identifyProcessedResponse.Remediation.RemediationOptions.FirstOrDefault().Method.Should().Be("POST");
+            identifyProcessedResponse.Remediation.RemediationOptions.FirstOrDefault().Accepts.Should().Be("application/ion+json; okta-version=1.0.0");
+
+            identifyProcessedResponse.Remediation.RemediationOptions.FirstOrDefault().Form.Should().NotBeNullOrEmpty();
+            var stateHandleFormItem = identifyProcessedResponse.Remediation.RemediationOptions.FirstOrDefault().Form.FirstOrDefault(x => x.Name == "stateHandle");
+            stateHandleFormItem.Should().NotBeNull();
+            stateHandleFormItem.Required.Should().BeTrue();
+            stateHandleFormItem.GetProperty<string>("value").Should().Be("02mOEmmIhklxzTn5W3erSQ0y9RwhjjPDHbvMTfgF5b");
+            stateHandleFormItem.Visible.Should().BeFalse();
+            stateHandleFormItem.Mutable.Should().BeFalse();
+
+            var authenticatorFormItem = identifyProcessedResponse.Remediation.RemediationOptions.FirstOrDefault().Form.FirstOrDefault(x => x.Name == "authenticator");
+            authenticatorFormItem.Should().NotBeNull();
+            authenticatorFormItem.Type.Should().Be("object");
+
+            authenticatorFormItem.Options.Should().NotBeNullOrEmpty();
+
+            authenticatorFormItem.Options.FirstOrDefault(x => x.Label == "Email").Should().NotBeNull();
+            var emailOption = authenticatorFormItem.Options.FirstOrDefault(x => x.Label == "Email");
+            emailOption.RelatesTo.Should().Be("$.authenticatorEnrollments.value[0]");
+
+            // TODO: Discuss mixed types for value with the team
```

Comment:
Heads up, tests on `OktaClientLayer1Should` aren't new. They were created when we were testing the "layer 1" (original strategy). The tests added by this PR are in `OktaClientShould`. 
That being said, what I'm calling out here is that we can't model `value` since it can be any type (array, object, etc.). This is still a concern that should be addressed if we want to follow a dynamic strategy.